src/utils/constants.js

```diff
@@ -2,8 +2,13 @@ const { LH_UI_VERSION } = process.env
 
 const C = {
   RegExp: {
-    // Regular expressions ensure that only the correct request path is included
-    REQUEST: /^(https|wss)?.+?(:\d{2,6})?(?=\/v1)/,
```

Comment:
This will replace `/v1/backupbackingimages/v1-xxx` with `v1-xxx`, causing the fetch URL to be incorrect.